### ISSUE TITLE
Gracefully handle Broken edge from Client

### ISSUE THREAD

**Issue by YaphetKG** [TranslatorSRI/Plater#31]
- Testing during relay Graceful error handling : 
a query with broken edge causes a time out in plater:
```{
   "message":{
      "query_graph":{
         "nodes":{
            "n0":{
               
            },
            "n1":{
               
            }
         },
         "edges":{
            "e0":{
               "subject":"uh oh",
               "object":"n1"
            }
         }
      }
   }
   ```
But it should return a 400 error with details about invalid edges

**Comment by YaphetKG** [comment]
Validators from reasoner pydantic are extended to treat this 

https://github.com/TranslatorSRI/Plater/blob/trapi-1.1/PLATER/models/shared.py

where base ReasonerRequest class is extended to validate its query attribute for the following assertions:
* Validates if atleast one node is bound else it won't run query (Linked to preventing "grab all" queries)
* Validates that edge node references are all present in the query_graph nodes keys

**Comment by YaphetKG** [comment]
@patrickkwang  this fix is part of #30
